### ISSUE TITLE
Contribution start point

### ISSUE THREAD

**Issue by tvbeek** [pdepend/pdepend#523]
## Description
We have no clear documentation for people who want to contribute. We could add a CONTRIBUTING.md and optional a page to the website.

**Comment by ravage84** [comment]
Some points to mentions/handle:

- We welcome contributors
- We actually are open for co-maintainers
- Code style guide (adherance to code standard?)
- Can the code be automatically formatted?
- How to test the code
- How to write tests
- How to write issues (checklist and/or issue templates)
- How to prepare a PR for success  (checklist and/or PR templates)
- Managring contributor expectations

**Comment by AJenbo** [comment]
> Can the code be automatically formatted?

Yes, see https://github.com/pdepend/pdepend/commit/4aa7fab1a7d35dd2845d772d8d0e06e51c704949
